src/App.tsx

```diff
@@ -1,10 +1,68 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { USER_ID } from './constants';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { ErrorType } from './types/Error';
+import { Filter } from './types/Filter';
+import { TodoList } from './components/TodoList';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isEdited] = useState(false);
+  const [isLoading, setLoading] = useState(false);
+  const [error, setError] = useState<ErrorType>(ErrorType.NONE);
+  const [filter, setFilter] = useState<Filter>(Filter.ALL);
+
+  const filterTodos = () => {
+    switch (filter) {
+      case Filter.ALL:
+      default:
```

Comment:
better move it to the end of the function

Reply:
Thanks, will do.